packages/api-client/src/account/AccountAPI.ts

```diff
@@ -30,6 +31,7 @@ export class AccountAPI {
     CALLS: '/calls',
     CALLS_CONFIG: 'config',
     CALLS_CONFIG_V2: 'v2',
+    CONFIG: '/config.json',
```

Comment:
Shall we name it `SSO_CONFIG` or `BACKEND_CONFIG`? We also introduced `CALLS_CONFIG` to not confuse things here.